### PR TITLE
feat(War Thunder): add presence

### PR DESCRIPTION
**Pull request by theusaf**
## Description 
Creates a new presence for War Thunder websites (wiki, tournaments, mods, etc.)

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/W/War Thunder/presence.ts [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+			if (pathname.startsWith("/feed/"))
+				presenceData.state = document.title.match(/(?:WT Live \/\/ )(.*)/)[1];
+			else if (pathname.startsWith("/post/")) {
+				presenceData.state = `Reading post: ${
+					document.title.match(/(?:WT Live \/\/ )(.*)/)[1]
+				}`;
+				presenceData.buttons = [
+					{
+						label: "View Post",
+						url: `https://${hostname}${pathname}`,
```
Comment (DarkVillager) [inline websites/W/War Thunder/presence.ts:79]: user href

File: websites/W/War Thunder/presence.ts [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+					presenceData.smallImageText = `Teams: ${
+						document.querySelector<HTMLSpanElement>("#all_teams_param_tour")
+							.textContent
+					} | Prize: ${
+						document.querySelector("[id-tss='prize_pool']").textContent
+					}`;
+					presenceData.buttons = [
+						{
+							label: "View Tournament",
+							url: `https://${hostname}${pathname}${search}`,
```
Comment (DarkVillager) [inline websites/W/War Thunder/presence.ts:119]: Use href

**Review by DarkVillager — CHANGES_REQUESTED** [review]
href?

File: websites/W/War Thunder/presence.ts [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+						.textContent,
+				action = searchParams.get("action");
+			if (action === "edit") presenceData.state = `Editing ${title}`;
+			else if (action === "history")
+				presenceData.state = `Viewing revision history of: ${title}`;
+			else presenceData.state = title;
+			break;
+		}
+	}
+	if (presenceData.details) presence.setActivity(presenceData);
```
Comment (DarkVillager) [inline websites/W/War Thunder/presence.ts:145]:
```suggestion
	if (presenceData.details) presence.setActivity(presenceData);
	else presence.setActivity();
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/W/War Thunder/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "theusaf",
+		"id": "193714715631812608"
+	},
+	"service": "War Thunder",
+	"description": {
+		"en": "Next-Gen MMO Combat Game for PC, Mac, Linux and PlayStation®4"
```
Comment (DarkVillager) [inline websites/W/War Thunder/dist/metadata.json:9]: image
```suggestion
		"en": "War Thunder is a free-to-play vehicular combat multiplayer video game developed and published by Gaijin Entertainment."
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**